WooCommerce/Classes/POS/Presentation/CouponRowView.swift

```diff
@@ -22,11 +24,29 @@ struct CouponRowView: View {
                 }
                 .frame(width: Constants.couponCardSize, height: Constants.couponCardSize)
 
-            VStack(alignment: .leading) {
+            VStack(alignment: .leading, spacing: Constants.itemTitleAndPriceSpacing * (1 / scale)) {
```

Comment:
Should we move this calculation to a computed variable? Something like `dynamicSpacing` or similar?